apps/condo/domains/contact/components/ContactsEditor/index.tsx

```diff
@@ -368,114 +449,49 @@ export const ContactsEditor: React.FC<IContactEditorProps> = (props) => {
-                                                                        onClick={handleClickOnPlusButton}
-                                                                        icon={<PlusCircleOutlined style={BUTTON_ICON_STYLE}/>}
-                                                                        disabled={disabled}
-                                                                    >
-                                                                        {AddNewContactLabel}
-                                                                    </Button>
-                                                                </Col>
-                                                            )
-                                                    )
-                                                }
-                                            </>
-                                        </>
-                                    )}
-                                </Row>
-                            </TabPane>
-                        )
-                    }
-                    {
-                        hasNotResidentTab && (
-                            <TabPane
-                                tab={TicketNotFromResidentMessage}
-                                key={CONTACT_TYPE.NOT_RESIDENT}
-                            >
-                                <Row gutter={TAB_PANE_ROW_GUTTERS}>
-                                    <NotResidentFields
-                                        initialQuery={initialEmployeesQuery}
-                                        refetch={refetchEmployees}
-                                        initialValue={initialValueWithoutContact}
-                                        onChange={handleChangeEmployee}
-                                        employees={fetchedEmployees}
-                                        activeTab={activeTab}
-                                    />
-                                </Row>
-                            </TabPane>
-                        )
-                    }
-                </Tabs>
+                {
+                    (canReadContacts || canManageContacts) && !hasNotResidentTab ? (
+                        <>
+                            <Typography.Title level={4}>{ResidentMessage}</Typography.Title>
+                            {residentInfo}
+                        </>
+                    ) : (
+                        <Tabs
+                            defaultActiveKey={activeTab}
+                            activeKey={activeTab}
+                            style={TABS_STYLE}
+                            onChange={handleTabChange}
+                            tabBarStyle={tabBarStyle}
+                        >
+                            {
+                                (canReadContacts || canManageContacts) && (
+                                    <TabPane tab={residentTitle || ResidentMessage} key={CONTACT_TYPE.RESIDENT} disabled={!unitName}>
```

Comment:
Looks like you should now update the props when using this component (If I'm not mistaken, this is only the ticket creation page) so that the texts are correct :)

Reply:
yes, updated it, thanks